grafana-plugin/src/containers/RotationForm/RotationForm.module.css

```diff
@@ -97,3 +96,21 @@
   margin-bottom: 10px;
   width: 100%;
 }
+
+@media (max-height: 768px) {
```

Comment:
Kind of a must because the scrollable container should scale differently based on window height (e. g. 45vh too less for anything higher than 1024px)